### PR TITLE
fix readme

### PR DESCRIPTION
**Pull request by eryajf**
https://github.com/lycheeverse/lychee-action/issues/144#issuecomment-1207366763

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ jobs:
 
       - name: Link Checker
         id: lychee
-        uses: lycheeverse/lychee-action@v1.5.0
+        # If you always want to use the latest features, you can replace the version lycheeverse/lychee-action@master
+        uses: lycheeverse/lychee-action@v1.5.1
```
Comment (tooomm) [inline README.md:37]: Not so sure about this comment. In the Readme it's explicitly advertised to pin versions and even use hashes: https://github.com/lycheeverse/lychee-action#security-and-updates
Using `@master` can also lead to unexpected breaking of the action.

Supporting "version branches" like other actions do, would at least prevent breaking changes: 
`@v1` (aka "give me the newest v1 minor/patch available with all new features and fixes without breaking changes")

Comment (eryajf) [reply README.md:37]: This is a good suggestion, but I tested the v1 version and found that the operation failed. I haven't found the reason yet.

Comment (tooomm) [reply README.md:37]: I think that needs to be enabled and supported specifically.
Which is not the case yet.

Comment (mre) [reply README.md:37]: Guess we have to push the v1 tag whenever we do a new release. We can do that at later point in time. For now I agree with @tooomm; let's remove the comment for the reasons he mentioned.

Comment (eryajf) [reply README.md:37]: Agree

**Comment by eryajf** [comment]
I temporarily removed the corresponding instructions and put the correct version into README. At least until better changes are made, what people see through README is available.

File: README.md (hunk 1 above)
Comment (eryajf) [reply README.md:36]: > Guess we have to push the v1 tag whenever we do a new release.

https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-release-management-for-actions

Comment (mre) [reply README.md:36]: Thanks for the link.

**Comment by mre** [comment]
FYI `v1` points to the latest `master` again.

**Comment by tooomm** [comment]
> FYI `v1` points to the latest `master` again.

Not sure you got it right. It shows v1.0.0 from December 2020 as the latest now. 🙈 

This is trickier than expected, haha.

**Comment by eryajf** [comment]
> Not sure you got it right. It shows v1.0.0 from December 2020 as the latest now. 🙈

I just ran a test with v1 and it worked normally. This is feasible

https://github.com/eryajf/learn-github/runs/7778974334?check_suite_focus=true

**Comment by tooomm** [comment]
It works, yes. But the [list of releases](https://github.com/lycheeverse/lychee-action/releases) is messed up now.

There must be another way when you look at other actions that support this.
Maybe a setting in the marketplace?

**Comment by tooomm** [comment]
It seems like there are some open questions or confusion still. @mre 

>FYI `v1` points to the latest `master` again.

`@v1` should not target master, but the newest `v1.*.*` release.
Not exactly sure, but I think there is `@master` to target master already?

I think it works for [peter-evans/create-issue-from-file](https://github.com/peter-evans/create-issue-from-file/releases), maybe @peter-evans himself can help with the correct marketplace and release setup to get moving releases like `v1` for the lychee-action as well? I did not find something useful via google. 🙈

Peter, once you release a `v4.0.1` version, you do tag it as such and everybody using `v4` should automatically start using it. Is there anything to setup in particular to make it work like that (marketplace, repo, releases...)?

**Comment by peter-evans** [comment]
I release using the GitHub releases UI and set the version to the full `vx.x.x` format. I then use git commands to move the major version tag from the previous release commit to the new release. The major version tag is not associated with any specific GitHub release. I manage it manually with git.

**Comment by mre** [comment]
Just a quick update: it worked. Thanks for the description @peter-evans.

Here's what I did for the latest release:
* Created a new tag through the Github releases UI
* Tagged v1 manually on the CLI: `git tag -f v1 v1.5.2`
* Pushed the new tag: `git push -f origin v1`

My original mistake was that I tagged `v1` to `master`, which had a newer commit than the latest point release. As a result it would show up on top of the list. Now that v1 and v1.5.2 point to the same commit v1 wasn't pushed to the top anymore.

**Pull request merged**